exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -448,39 +427,44 @@ public Builder withValidators(short validatorCount) {
     }
 
     /**
-     * Adds a service with which the TestKit would be instantiated. Several services can be added.
+     * Adds a service artifact which would be deployed by the TestKit. Several service artifacts
+     * can be added.
+     *
+     * <p>Note that the corresponding service instance with equal serviceArtifactId should be
```

Comment:
```
Once the service artifact is deployed, the service instances can be added with
{@link #withService(ServiceArtifactId, String, int, Any)}.
```

Reply:
I'm not sure about the new description. After invoking `withDeployedArtifact()` the artifact isn't immediately deployed, so it sounds a bit confusing.